src/templates/specific-project.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react'
+import { Link, graphql } from 'gatsby'
+import Img from 'gatsby-image'
+import Layout from '../components/Layout.js'
+import Navbar from '../components/Navbar.js'
+import { Row, Col } from 'antd';
+import { Button } from 'antd';
+
+const ProjectTemplate = ({ data }) => (
+    <Layout>
+        <br></br>
+        <br></br>
+        <br></br>
+        <Row type="flex" justify="center" align="center" >
+            <Col xs={20} sm={10} m={10} l={12} xl={6}>
```

Comment:
These sections have overlap problems...

![Screenshot 2019-03-18 at 8 24 19 PM](https://user-images.githubusercontent.com/42818904/54578173-ee770400-49bb-11e9-8a79-4ccc38471d49.png)

Reply:
> I'm open to merging this as-is, but only if someone adds tasks to the backlog to address all of my comments (I don't see any in trello or the issues list at the moment).

I added all of these tasks to the backlog on trello. They're on a checklist on the "Specific Project Page fixes" ticket